cmd/config-current.go

```diff
@@ -663,6 +651,24 @@ func applyDynamicConfig(ctx context.Context, objAPI ObjectLayer, s config.Config
 		return fmt.Errorf("Unable to apply scanner config: %w", err)
 	}
 
+	// Logger webhook
+	loggerCfg, err := logger.LookupConfig(s)
+	if err != nil {
+		logger.LogIf(ctx, fmt.Errorf("Unable to load logger webhook config: %w", err))
+	}
+	userAgent := getUserAgent(getMinioMode())
+	for _, l := range loggerCfg.HTTP {
+		if l.Enabled {
+			l.LogOnce = logger.LogOnceIf
+			l.UserAgent = userAgent
+			l.Transport = NewGatewayHTTPTransportWithClientCerts(l.ClientCert, l.ClientKey)
+		}
+	}
+	err = logger.UpdateTargets(loggerCfg)
+	if err != nil {
+		logger.LogIf(ctx, fmt.Errorf("Unable to update logger webhook config: %w", err))
+	}
+
```

Comment:
@anjalshireesh existing targets that are running are not canceled here so - you need to add that as well

Reply:
@harshavardhana Aren't they replaced?

Reply:
> @harshavardhana Aren't they replaced?

They run in a goroutine - I don't remember ever writing a canceler.

Reply:
@harshavardhana I've tried to implement canceler for the logger webhook targets. PTAL.